dev/PowerNotifications/PowerNotifications.h

```diff
@@ -666,7 +666,7 @@ namespace winrt::Microsoft::ProjectReunion::implementation
             return make_self<factory_implementation::PowerManager>()->EnergySaverStatusChanged_Callback(energySaverStatus);
         }
 
-        static void CompositeBatteryStatusChanged_Callback(CompositeBatteryStatus* compositeBatteryStatus)
+        static void CompositeBatteryStatusChanged_Callback(CompositeBatteryStatus compositeBatteryStatus)
```

Comment:
Just because we're changing the type to CompositeBatteryStatus* for outparams doesn't mean it needs to be CompositeBatteryStatus for in-params. If you do that, you're copying the struct for the function calls. I'd say go with either CompositeBatteryStatus* or CompositeBatteryStatus& to avoid unnecessary copying.

Also: we should make a point of putting const on all of the places where in-params are being passed by reference/pointer.